README.md

```diff
@@ -27,20 +27,20 @@ $ git clone --depth=1 https://github.com/juristr/webpack-typescript-starter.git
 ```
 Maybe remove the `.git` directory and start with a fresh one.
 
-run `npm i` to install all dependencies.
+run `$ npm i` to install all dependencies.
```

Comment:
I'd remove the `$` here. Usually if I mention commands inline I don't add it as prefix. Rather I do it when having multi-line code which represents terminal output like

```
$ echo Hi
Hi
```

:smile: